src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -30,13 +37,27 @@ export function flatten(data) {
 
 export function transform(formConfig, form) {
   const formData = flatten(transformForSubmit(formConfig, form));
+  delete formData.storePrefillStatus;
```

Comment:
I think you'd want to delete formData.prefilled now?

Reply:
Correct, fixed.